doc/Homepage.ipynb

```diff
@@ -17,7 +17,7 @@
    "source": [
     "import numpy as np\n",
     "import holoviews as hv\n",
-    "%reload_ext holoviews.ipython\n",
+    "hv.notebook_extension('matplotlib')\n",
```

Comment:
It is nice to put ``'matplotlib'`` in there even it isn't necessary to emphasis we support multiple backends. What would you think about putting a comment here...maybe along the lines of ``# Or 'bokeh' if available``?